runtime/src/lib.rs

```diff
@@ -593,6 +593,8 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = ();
 	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+	type ControllerOrigin = EnsureRoot<AccountId>;
```

Comment:
@riusricardo I'm not sure when we should use `EnsureRoot` and when we should use `EnsureOneOf<EnsureRoot<AccountId>, EnsureXcm<IsMajorityOfBody<RelayLocation, ExecutiveBody>>>;` (so also allowing the relay chain council). Wdyt?

Reply:
This is just a security measure to disable XCM. But we don't have sudo in Canvas so this functionality will not work unless we add the EnsureXcm to handle it from the relay chain.

This opens the question of adding a second runtime with sudo for a much easier and faster development.

Reply:
> This opens the question of adding a second runtime with sudo for a much easier and faster development.

There's gotta be a better way to tune this one setting than to add the overhead of a second runtime 😄

Reply:
@riusricardo so which of both do we want. I think for common good we want `EnsureOneOf<EnsureRoot<AccountId>, EnsureXcm<IsMajorityOfBody<RelayLocation, ExecutiveBody>>>;`, right?

Reply:
Alright, I've updated it. Could someone approve the PR plz?